packages/showcases/css/stories/components/navigation/tabs/examples/demo.html

```diff
@@ -14,8 +13,63 @@
       <li id="tab-demo-4" role="tab" aria-selected="false">
         <a href="">Tab Name</a>
       </li>
-      <span id="block-focus-trap-end" tabindex="-1"></span>
     </ul>
     <div class="vtmn-divider" role="separator"></div>
   </nav>
 </div>
+
+<script>
+  let listTabs = document.querySelectorAll('#demo-tabs-tablist li > a');
+  let nav = document.getElementById('demo-tabs-nav');
+
+  listTabs.forEach((child, index, parent) => {
+    child.addEventListener('keydown', (e) => {
+      switch (e.key) {
+        case 'ArrowLeft':
+          if (index <= 0) {
+            listTabs[listTabs.length - 1].focus();
+          } else {
+            listTabs[index - 1].focus();
+          }
+          break;
+        case 'ArrowRight':
+          if (index >= listTabs.length - 1) {
+            listTabs[0].focus();
+          } else {
+            listTabs[index + 1].focus();
+          }
+          break;
+        case 'Home':
+          listTabs[0].focus();
+          break;
+        case 'End':
+          listTabs[listTabs.length - 1].focus();
+          break;
+        case 'Tab':
+          e.preventDefault();
+          if (e.shiftKey) {
+            console.log('shift');
+            listTabs[0].focus();
+            listTabs[0].blur();
+          } else {
+            console.log('no shift');
+            listTabs[listTabs.length - 1].focus();
+            listTabs[listTabs.length - 1].blur();
+          }
```

Comment:
According to the[ W3C accessibility tabs pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/) you shared: 

> _When the tab list contains the focus, moves focus to the next element in the page tab sequence outside the tablist, which is the tabpanel unless the first element containing meaningful content inside the tabpanel is focusable._

So arrows should be used to navigate the tablist, but tab key should be used to enter inside the tabpanel.

To do that you could set all tablist elements (tab) to a tabindex of -1, except for the active tab which should remain focusable. This way the next focusable element would be the first visible tabpanel.

I'll share with you a demo if you want

Reply:
In our case we don't have a tabpanel associated with the tabs. Do you think we should adda tabpanel ? In this case we would need a design template for that as it is not included in the basic composant